src/OrchardCore.Modules/OrchardCore.Contents/Permissions.cs

```diff
@@ -44,7 +46,9 @@ public Task<IEnumerable<Permission>> GetPermissionsAsync()
                 PreviewContent,
                 CloneContent,
                 CloneOwnContent,
-                AccessContentApi
+                AccessContentApi,
+                AccessAdminContentList,
```

Comment:
Less sure if we need this extra permission. Does it add more confusion, or less?

Reply:
The other option would be to evaluate if a user has access to any content. This would be a little more intense than simply allowing or not access to that list. Before we we're allowing users with the Permission.EditContent but this is a permission that allows editing every content type (too global). I made sure that this permission is added by default for the roles that used to have access to that list. So, if you never had to restrict per content type you will maybe be confused but this is adding more fine tuned access level for people.